superset/sql_parse.py

```diff
@@ -75,32 +75,32 @@ def get_statements(self):
         return statements
 
     @staticmethod
-    def __get_full_name(identifier):
-        if len(identifier.tokens) > 2 and identifier.tokens[1].value == '.':
-            return '{}.{}'.format(identifier.tokens[0].value,
-                                  identifier.tokens[2].value)
-        return identifier.get_real_name()
+    def __get_full_name(tlist: TokenList):
+        if len(tlist.tokens) > 2 and tlist.tokens[1].value == '.':
+            return '{}.{}'.format(tlist.tokens[0].value,
+                                  tlist.tokens[2].value)
+        return tlist.get_real_name()
 
     @staticmethod
-    def __is_identifier(token):
+    def __is_identifier(token: Token):
         return isinstance(token, (IdentifierList, Identifier))
 
-    def __process_identifier(self, identifier):
+    def __process_tokenlist(self, tlist: TokenList):
```

Comment:
Calling this `identifiers` is somewhat misleading as in reality it's processing `TokenList` objects.